helpers.js

```diff
@@ -426,6 +426,11 @@ function createStreamInfo(item, mediaSource, startPosition) {
     var subtitleStreams = mediaSource.MediaStreams.filter(function (stream) { return stream.Type === "Subtitle"; });
     var subtitleTracks = []
     subtitleStreams.forEach(function(subtitleStream) {
+        if(subtitleStream.Codec !== 'vtt' && subtitleStream.Codec !== 'webvtt') {
```

Comment:
```suggestion
        if (subtitleStream.Codec !== 'vtt' && subtitleStream.Codec !== 'webvtt') {
```

Reply:
@YouKnowBlom Might be worth adding some toLower here in case it changes casing at some point.

Reply:
Good catch, I'll add that